mousebender/simple.py

```diff
@@ -1,13 +1,12 @@
 """Parsing for PEP 503 -- Simple Repository API."""
-from __future__ import annotations
-
-import dataclasses
+import collections
```

Comment:
What's this for?

Reply:
Not a thing. Added it when I was messing around with 3.5 support.